Sources/ApodiniDeploy/ApodiniDeployInterfaceExporter.swift

```diff
@@ -145,54 +133,50 @@ class ApodiniDeployInterfaceExporter: LegacyInterfaceExporter {
     
     
     private func performDeploymentRelatedActions() throws {
+        try self.exportDeployedSystemIfNeeded()
+        
         let env = ProcessInfo.processInfo.environment
-        guard let mode = env[WellKnownEnvironmentVariables.executionMode],
-              let fileURL = env[WellKnownEnvironmentVariables.fileUrl]
-        else {
+        
+        let configUrl: URL
+        let currentNodeId: String
+        
+        let deployedSystem: AnyDeployedSystem
+        
+        // The aws deployment provider still uses env variables,
+        // so first check if there are any present
+        if env[WellKnownEnvironmentVariables.fileUrl] != nil,
+           env[WellKnownEnvironmentVariables.currentNodeId] != nil {
```

Comment:
What speaks agains moving them to command line arguments in the startup command? As far as I can see they should be usable in that way and are only used on startup as far as I can see.

Reply:
I reason i added this was that as far as I remember there were some down sides to not using environment variables when we talked about the refactoring a couple of weeks ago. Maybe @lukaskollmer has some input on that. I'm still no super familiar with the aws deployment provider.

Reply:
Ah, I remember that this was due to the fact that this would require multiple version of the web service being uploaded to the S3 storage as the command line arguments are fixed and can not be changed for each Lambda Startup with a `.zip` file, only Environment variables can be changed.

As far as I remember we just considered adding a small shell script or something that passes these environment variables to the command line arguments or explore if it is possible to directly reference environment variables in the startup command for Lambda functions to eliminate them here and make it easier to test this locally as you only have to rely on the CLI interface and do not have to also remember and set environment variables too.

Reply:
@PSchmiedmayer you're right, the lambdas require these values be passed via the environment if there is to be only one zip file.
You're also right in that it should be possible to, as part of the lambdas' `boostrap` program, read these from the environment and set them as CLI params; I'll see if I can implement that into this PR, that shouldn't be too much work.

Reply:
Great to hear @lukaskollmer, thanks!